packages/cli/src/ui/commands/aboutCommand.ts

```diff
@@ -31,7 +32,7 @@ export const aboutCommand: SlashCommand = {
     const gcpProject = process.env['GOOGLE_CLOUD_PROJECT'] || '';
     const ideClient =
       (context.services.config?.getIdeMode() &&
-        context.services.config?.getIdeClient()?.getDetectedIdeDisplayName()) ||
+        (await IdeClient.getInstance())?.getDetectedIdeDisplayName()) ||
```

Comment:
In general I'd avoid inlining `await` clauses because they aren't very debbugable